### ISSUE TITLE
Enhancement: Generalized Linear Mixed Models

### ISSUE THREAD

**Issue by kornbrot** [jasp-stats/jasp-desktop#2485]
* Enhancement: Generalized Linear Mixed Models, GLMM
* Purpose: Enable repeated measures for proportions (binomial), counts (Poisson). Include Bayes Information Criteria, BIC. Same facilities as SPSS Mixed.
Know its a tough ask, BUT
we have submitted Mss that show

1.  ANOVA and GLMM give different answers for both inferential and descriptive stats
2. BIC reliable discriminates among link functiions

**Comment by lindeloev** [comment]
This was also requested in #1103 and #1846, for the record. Jamovi offers a nice implementation of mixed models and GLM though not GLMM.

**Comment by EJWagenmakers** [comment]
Yes, good suggestions and on our radar since the inception of JASP.
